types/office-js/index.d.ts

```diff
@@ -73791,6 +75084,14 @@ declare namespace Excel {
              * [Api set: ExcelApi 1.8]
              */
             formula?: string;
+            /**
+             * Gets or sets the geometric shape type of the data label. See `Excel.GeometricShapeType` for more details.
```

Comment:
Should these "Gets or sets" say "Specifies" instead?

```suggestion
             * Specifies the geometric shape type of the data label. See `Excel.GeometricShapeType` for more details.
```

Reply:
I think "get or sets" is fine—there are 118 instances of "gets or sets" in this file throughout the Excel and other namespaces.